### PR TITLE
chore(example): Rename files to Dart style

### PR DESCRIPTION
**Pull request by viveky259259**
Issue #, if available:

Description of changes:
I found that file naming was not consistent in example/lib folders.
So, I updated the file names and their respecting imports.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: example/lib/main.dart [hunk 1]
```diff
@@ -18,10 +18,9 @@ import 'package:amplify_auth_cognito/amplify_auth_cognito.dart';
 import 'package:amplify_flutter/amplify_flutter.dart';
 import 'package:amplify_storage_s3/amplify_storage_s3.dart';
 import 'package:flutter/material.dart';
-import 'package:sample_app/Pages/LoadingPage.dart';
-
-import 'Pages/LandingPage.dart';
+import 'package:sample_app/Pages/loading_page.dart';
 
+import 'Pages/landing_page.dart';
```
Comment (Jordan-Nelson) [inline example/lib/main.dart:23]: The Dart analyzer won't complain about the mismatch in casing, but can you update it to match the new casing?

```suggestion
import 'pages/landing_page.dart';
```

File: example/lib/pages/landing_page.dart [hunk 2]
```diff
@@ -15,9 +15,9 @@
 
 import 'package:flutter/material.dart';
 
-import '../Views/SignInView.dart';
-import '../Views/SignUpView.dart';
-import 'MainPage.dart';
+import '../Views/sign_in_view.dart';
+import '../Views/sign_up_view.dart';
```
Comment (Jordan-Nelson) [inline example/lib/pages/landing_page.dart:19]:
```suggestion
import '../views/sign_in_view.dart';
import '../views/sign_up_view.dart';
```

File: example/lib/pages/main_page.dart [hunk 3]
```diff
@@ -16,9 +16,9 @@
 import 'package:amplify_flutter/amplify_flutter.dart';
 import 'package:amplify_storage_s3/amplify_storage_s3.dart';
 import 'package:flutter/material.dart';
-import 'package:sample_app/Views/ImageLineItem.dart';
-import 'package:sample_app/Views/ImageUploader.dart';
-import 'package:sample_app/Views/UserView.dart';
+import 'package:sample_app/Views/image_line_item.dart';
+import 'package:sample_app/Views/image_uploader.dart';
+import 'package:sample_app/Views/user_view.dart';
```
Comment (Jordan-Nelson) [inline example/lib/pages/main_page.dart:21]:
```suggestion
import 'package:sample_app/views/image_line_item.dart';
import 'package:sample_app/views/image_uploader.dart';
import 'package:sample_app/views/user_view.dart';
```

File: example/lib/views/sign_up_view.dart [hunk 4]
```diff
@@ -16,7 +16,7 @@
 import 'package:amplify_auth_cognito/amplify_auth_cognito.dart';
 import 'package:amplify_flutter/amplify_flutter.dart';
 import 'package:flutter/material.dart';
-import 'package:sample_app/Views/ErrorView.dart';
+import 'package:sample_app/Views/error_view.dart';
```
Comment (Jordan-Nelson) [inline example/lib/views/sign_up_view.dart:19]:
```suggestion
import 'package:sample_app/views/error_view.dart';
```

File: example/lib/views/user_view.dart [hunk 5]
```diff
@@ -15,7 +15,7 @@
 
 import 'package:amplify_flutter/amplify_flutter.dart';
 import 'package:flutter/material.dart';
-import 'package:sample_app/Pages/LandingPage.dart';
+import 'package:sample_app/Pages/landing_page.dart';
```
Comment (Jordan-Nelson) [inline example/lib/views/user_view.dart:18]:
```suggestion
import 'package:sample_app/pages/landing_page.dart';
```

**Review by Jordan-Nelson — COMMENTED** [review]
Thanks for opening the PR. I left a few suggested changes, but otherwise looks good.

File: example/lib/pages/landing_page.dart (hunk 2 above)
Comment (viveky259259) [reply example/lib/pages/landing_page.dart]: Done

File: example/lib/views/sign_up_view.dart (hunk 4 above)
Comment (viveky259259) [reply example/lib/views/sign_up_view.dart]: Done

**Comment by viveky259259** [comment]
@Jordan-Nelson  I have done suggested changes

**Review by dnys1 — APPROVED** [review]
Thanks @viveky259259! Pushed a small change to keep `ModelProvider.dart` as-is. Unfortunately, this is how the Amplify CLI outputs it, so this keeps things consistent with that.

Otherwise, looks great. Thanks for the PR!

File: example/lib/main.dart [hunk 6]
```diff
@@ -33,7 +32,7 @@ You will use the Amplify CLI tool for that.  Please read the
 README.md contained within the root of this project to learn
 about what you'll need to do.
  */
-import 'amplifyconfiguration.dart';
+// import 'amplifyconfiguration.dart';
```
Comment (Jordan-Nelson) [inline example/lib/main.dart:35]: Let's keep this as is. As the comment above notes, to run the example, a configuration file is needed. Commenting this out will not remove need to generate a config file, so it really just introduces an extra step to uncomment this.
```suggestion
import 'amplifyconfiguration.dart';
```

File: example/lib/main.dart [hunk 7]
```diff
@@ -62,7 +61,7 @@ class _MyAppState extends State<MyApp> {
 
     // Initialize AmplifyFlutter
     try {
-      await Amplify.configure(amplifyconfig);
+      await Amplify.configure('amplifyconfig');
```
Comment (Jordan-Nelson) [inline example/lib/main.dart:64]: Same as above. Let's keep this as is.
```suggestion
      await Amplify.configure(amplifyconfig);
```

**Review by Jordan-Nelson — APPROVED** [review]
Thanks for the PR.

I added a commit to undo the changes to comment out `import 'amplifyconfiguration.dart';` in main.dart. We intentionally do not commit this to source control, but the example app requires generating a config. Commenting out the import will add an extra step each time the app is run locally. Let me know if you have any questions.

**Comment by viveky259259** [comment]
thanks @Jordan-Nelson

**Review by dnys1 — APPROVED** [review]

**Comment by viveky259259** [comment]
Also, regarding, `ModelProvider.dart`,  can we generate code with the file `model_provider.amplify.dart`?
Here `model_provider` is the name of the file, and `.amplify` will say that it was auto-generated by amplify SDK.

**Pull request merged**